sktime/classification/__init__.py

```diff
@@ -2,6 +2,8 @@
 
 __all__ = [
     "BaseClassifier",
+    "OSCNNClassifier"
 ]
 
 from sktime.classification.base import BaseClassifier
+from sktime.classification._oscnn import OSCNNClassifier
```

Comment:
we should not add the export here

Reply:
@fkiraly hi, could you please reference me to where I should add this? should it be added to the `base.py ` file?